.github/workflows/lint.yml

```diff
@@ -21,6 +21,9 @@ jobs:
     runs-on: ubuntu-latest
     container:
       image: ghcr.io/realm/swiftlint:0.50.3
+      credentials:
+        username: ${{ github.actor }}
+        password: ${{ secrets.github_token }}
```

Comment:
This fixes and issue where the swiftlint action was failing due to an authentication error from the github container registry.

Reply:
weird, why would this container start requiring credentials now?
Maybe a change by SwiftLint team?